src/main/cluster.ts

```diff
@@ -133,7 +133,13 @@ export class Cluster implements ClusterModel {
     if (this.disconnected || (!init && !this.accessible)) {
       await this.reconnect();
     }
-    await this.refresh();
+    await this.refreshConnectionStatus()
+    if (this.accessible) {
+      await this.refreshAllowedResources()
+      this.ready = true
+      this.kubeCtl = new Kubectl(this.version)
+      this.kubeCtl.ensureKubectl() // download kubectl in background, so it's not blocking dashboard
```

Comment:
But still, will it block sending the state via `pushState` below? If so that means dashboard won't receive "ready-state" until downloading finished?

Reply:
I tested by removing kubectl binaries from app resources and did not see the same empty screen as with master  anymore when opened the dashboard.

Reply:
Oh, right, there is no `await` for `kubeCtl.ensureKubectl()`